unattended_scripts/install_functions/opendistro/wazuh-passwords-tool.sh

```diff
@@ -212,6 +284,17 @@ generatePassword() {
  
 }
 
+generatePasswordFile() {
+    USERS=(admin kibanaserver kibanaro logstash readall snapshotrestore wazuh_admin wazuh_user)
```

Comment:
This should not be hardcoded

Reply:
This function generates the passwords for the users present in a default installation and can be run in a machine that does not have ES installed so it has to know all the users beforehand.

Reply:
The internal_users.yml file that provides these hardcoded users may vary in time. Let's imagine that an user adds a new user before performing the installation using the --local parameter, that user would not be reflected and the password for that user would not change. Another case, in the future, we could delete or add a new user, then we will have to find this line and change it. The users should be obtained by reading the file and storing them in a variable.